lib/homePage.dart

```diff
@@ -51,6 +53,19 @@ class HomePageView extends State<HomePage> with SingleTickerProviderStateMixin {
     _animationController?.stop();
     _animationController?.dispose();
     super.dispose();
+    WidgetsBinding.instance.removeObserver(this);
+  }
+
```

Comment:
Could you just add comments here so that it would be easy to understand what this section is for others. You might know this dart page is pretty complex.

Reply:
Ok, I will add the comments, Thanks for the suggestion.